README.md

```diff
@@ -40,6 +40,8 @@ the &lt;TAB&gt; key to move to the next placeholder. All FSH snippets always sto
 | `map`   | Mapping    | Mapping, Source, Target, Id, Title (auto), Description          |
 | `rs `   | RuleSet    | RuleSet                                                         |
 
+Additionally, a snippet is provided to help when writing slicing rules. When the phrase `^sli` is detected, a block of rules can be inserted for the paths commonly set when defining slicing on an element.
```

Comment:
I'm seeing that the snippet works even when you only type `^s`. Is that intentional? If so, maybe these docs should be updated.